virttest/qemu_vm.py

```diff
@@ -4300,11 +4300,19 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
             # If we're doing remote migration and it's completed successfully,
             # self points to a dead VM object
             if not not_wait_for_migration:
-                if self.is_alive() and self.is_paused():
+                if self.is_alive():
                     # For short period of time the status can be "inmigrate"
                     # for example when using external program
                     # (qemu commit fe823b6f87b2ebedd692ca480ceb9693439d816e)
-                    self.resume(60)
+                    # resume vm during "inmigrate" status can't work, wait until
+                    # exit "inmigrate" status and resume vm if needed.
+                    if utils_misc.wait_for(lambda: not self.monitor.verify_status('inmigrate'),
+                                           timeout=120, step=0.1):
+                        if self.is_paused():
+                            self.resume()
+                    else:
+                        raise virt_vm.VMStatusError("vm can't exit 'inmigrate' status"
+                                                    "after 120s")
```

Comment:
It'd be probably useful to log the status here, but it's not that important as we can get it from the monitor logs...

Reply:
@ldoktor From the error info in debug.log: `can't exit 'inmigrate' status`, it means the status is "inmigrate" after 120s.
So i think we do not need to log `self.monitor.get_status()` again. Thanks!